src/Controller/DashboardController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of monofony.
+ *
+ * (c) Mobizel
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Dashboard\DashboardStatisticsProvider;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+
+class DashboardController extends AbstractController
```

Comment:
As we are using proper DI in this controller I would vote for removing AbstractController inheritance and just inject twig renderer. 

Also, we could finalize this service. Ref. http://ocramius.github.io/blog/when-to-declare-classes-final/

Reply:
Same for all of services